armoryengine/Networking.py

```diff
@@ -21,14 +21,15 @@
 
```

Comment:
ignore all changes in Networking.py, all p2p messaging is handled in cppForSwig/BitcoinP2p.h/cpp now